### ISSUE TITLE
Sidebar render issue

### ISSUE THREAD

**Issue by erossini** [docsifyjs/docsify#1836]
I'm creating a documentation about Agile. I have an interesting issue with the sidebar. For example, this is the Markdown I use for the Scrum section

```
* [Scrum](./scrum/intro.md)
  - [3 pillars](./scrum/pillars.md)
  - [Scrum team](./scrum/scrum-team.md)
    - [Product owner](./scrum/product-owner.md)
    - [Development team](./scrum/development-team.md)
    - [Scrum master](./scrum/scrum-master.md)
  - [Sprint events](./scrum/scrum-events.md)
    - [Sprint](./scrum/sprint.md)
    - [Sprint planning](./scrum/sprint-planning.md)
    - [Daily Scrum](./scrum/daily-scrum.md)
    - [Sprint Review](./scrum/sprint-review.md)
    - [Sprint retrospective](./scrum/sprint-retrospective.md)
  - [Artifacts](./scrum/artifacts.md)
    - [Product backlog](./scrum/product-backlog.md)
    - [Sprint backlog](./scrum/sprint-backlog.md)
    - [Increment](./scrum/increment.md)
  - [Advantages and disadvantages](./scrum/pros-cons.md)
```

the render for this sidebar is as the following image

![image](https://user-images.githubusercontent.com/9497415/178219917-55d3ef0c-5424-45e8-bd20-44294271e691.png)

I expect to see **Scrum team** with an arrow to see the details. Instead, I see immediately the submenu with a `-` like `- Product owner` under if I click on the arrow, there is no submenu for this option.

In the configuration, I added

```
      loadSidebar: true,
      maxLevel: 5,
      subMaxLevel: 5,
```

How can I fix this? 

Thank you in advance,
Enrico

**Comment by sy-records** [comment]
If `. /scrum/scrum-team.md` exists `h1`... etc. headings, it renders `-`

**Comment by jhildenbiddle** [comment]
This issue has been addressed by PR #2469. The changes will be available when Docsify v5 is released.
